src/views/guidelines/guidelines.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../colors";
+@import "../../typography";
+
+.view {
+    .guidelines-footer {
```

Comment:
I think it might be good to add some margins between the `dt/dd` tags, so that it looks like this:
```
#view {
    dl {
        dt {
            margin-top: 1.5rem;
        }
    }

    .guidelines-footer {
        margin-top: 1.5rem;
        text-align: center;
    }
}
```
@TheGrits If you could make those slight changes, I think this is ready to go!